### ISSUE TITLE
Can't place ore blocks

### ISSUE THREAD

**Issue by djchena** [WhiskyTangoFawks/Expedition#34]
I can't place any ore block (included mined with silk touch), only configuring WTFGameplay.cfg to false in WTFCore.cfg. I try to modify many WTFGameplay options, but nothing.

**Comment by WhiskyTangoFawks** [comment]
That's working as intended.  It's part of the ore's fracture adjacent stone feature in the gameplay module.

Disable the gameplay module in WTFCore.cfg, or "disable ores fracture adjacent stone" in WTFGameplay.cfg

**Comment by charlesbevans** [comment]
Out of curiosity, is there a reason? I can throw up some smooth stone if I want to go fracturing things, and the inability to place ore interferes with some mods, like EU2's redstonia plant.

**Comment by WhiskyTangoFawks** [comment]
Smooth stone doesn't fracture adjacent, or at least it shouldn't.

Also- smooth touch shouldn't be working on the dense ore blocks- you shouldn't be able to get the dense versions of the block outside of creative mode, it should always drop the vanilla block.

**Comment by charlesbevans** [comment]
Oh yeah, I made smooth and decorative stone fracture adjacent stone to make mining more perilous. :) Lot of fun. I wonder if chisel adds cobble versions of decorative stone...
I was just curious from a design point. I realize now it was to prevent speeding up mining by fracturing stone to cobble, which I had turned off. (It was a bit too slow and the difference with non-smooth stone was considerable. Besides, I could always "cheat" with a chisel.)

**Comment by WhiskyTangoFawks** [comment]
Not what I intended, but if you're enjoying it and it's working, nothing wrong with that.

**Comment by charlesbevans** [comment]
It was great. My mining speed was consistent. I constantly evaluated the stone around me, and still got hit sometimes. The only real problems were the number of fence posts I needed, and when I mined out a room, it tended to form a checkerboard pattern of smooth stone and cobblestone.
Again, thanks. I'm working on an underground dwarfy pack (using the Underground terrain gen mod) and Expedition is providing decoration and custom ore gen. I've turned off the fracturing for now. I think stable cobblestone will be important.
